0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -2,7 +2,7 @@
 coding: utf-8
 
 title: Crypto-Conditions
-docname: draft-thomas-crypto-conditions-00
+docname: draft-thomas-crypto-conditions-02
```

Comment:
The first version must be -00, which the IETF tracker helpfully forced me to abide by when I uploaded it. Ignore https://interledger.org/five-bells-condition/spec.html - that link is deprecated and was never authoritative. Internet Drafts are tracked by the IETF and the current version number is -00:

https://tools.ietf.org/html/draft-thomas-crypto-conditions-00

So the next version we upload has to be -01.